src/editorial/web/components/logo/index.tsx

```diff
@@ -0,0 +1,147 @@
+					fill="#fff"
+				/>
+				<path
+					d="M180.622 58.804v-11.57l-4.131-1.443v-.924l15.213-2.791 1.447.201v49.686l4.205 1.518v1.284l-15.01 2.016-1.171-.137v-4.108h-.34c-2.204 2.197-5.238 4.182-9.986 4.182-8.197 0-14.18-6.241-14.18-19.01 0-13.46 6.952-20.072 17.491-20.072 3.013 0 5.291.552 6.462 1.168zm-.032 31.81V60.917c-.969-.616-1.661-1.38-4.163-1.295-4.066.138-6.578 6.273-6.578 17.184 0 9.819 1.809 15.306 7.228 15.126 1.522-.053 2.757-.595 3.513-1.316zM214.07 57.848l1.309.138V92.46l3.439 1.9v1.4h-19.545v-1.39l3.513-1.9v-29.04l-4.205-1.645v-1.147l15.489-2.791zm1.384-9.31c0 3.642-3.098 6.38-6.675 6.38-3.715 0-6.611-2.749-6.611-6.38 0-3.64 2.896-6.453 6.611-6.453 3.577 0 6.675 2.813 6.675 6.454zM261.453 92.472v-29.36l-4.131-1.443v-1.422l15.01-2.792 1.512.138v4.394h.415c3.236-2.887 8.059-4.734 12.807-4.734 6.536 0 9.432 3.09 9.432 9.957v25.198L300 94.36v1.4h-19.545v-1.39l3.513-1.9V67.9c0-3.778-1.65-5.285-4.748-5.285-2.001 0-3.641.51-5.163 1.634v28.213l3.438 1.9v1.4H257.94v-1.39l3.513-1.9zM239.768 74.024v-4.925c0-7.42-1.618-9.85-6.217-9.85-.543 0-1.011.063-1.554.137l-8.186 11.06h-1.15V60.269c3.513-1.083 7.91-2.357 13.733-2.357 10.006 0 15.829 2.77 15.829 11.124v24.01l3.588.944v.945c-1.416.88-4.258 1.687-7.377 1.687-4.94 0-7.303-1.613-8.389-4.32h-.341c-2.097 2.834-5.067 4.448-9.74 4.448-5.951 0-10.007-3.705-10.007-10.116 0-6.209 3.854-9.574 11.699-11.06l8.112-1.55zm0 16.59v-14.84l-2.501.203c-3.929.34-5.344 2.834-5.344 8.364 0 5.997 1.958 7.557 4.737 7.557 1.554-.01 2.438-.478 3.108-1.284zM110.202 74.024v-4.925c0-7.42-1.618-9.85-6.227-9.85-.543 0-1.012.063-1.555.137l-8.186 11.06h-1.15V60.269c3.513-1.083 7.91-2.357 13.733-2.357 10.007 0 15.83 2.77 15.83 11.124v24.01l3.587.944v.945c-1.416.88-4.258 1.687-7.377 1.687-4.94 0-7.303-1.613-8.389-4.32h-.34c-2.097 2.834-5.078 4.448-9.741 4.448-5.95 0-10.007-3.705-10.007-10.116 0-6.209 3.854-9.574 11.7-11.06l8.122-1.55zm0 16.59v-14.84l-2.501.203c-3.929.34-5.344 2.834-5.344 8.364 0 5.997 1.958 7.557 4.737 7.557 1.543-.01 2.427-.478 3.108-1.284z"
+					fill="#fff"
+				/>
+			</svg>
```

Comment:
Sorry to ask, but what is stopping us using the existing logo SVG `<SvgGuardianLogo>`? Is it because the viewbox is different to the other logos, and hence it causes issues with height and width? 

If so, is it possible to specify only `width` for `svgSize` and allow `height` to be calculated dynamically? Or the other way around? I'm just a bit wary of having two sources of truth for logo SVGs.

Reply:
Maybe I misunderstood your request, I thought you wanted them all to have the same size viewbox.
I think just specifying a px value for width and then height:auto would help us keep consistency here(thats how we currently are scaling them in the DCR header). If any future logo variants always have "The Guardian" as the widest element in the SVG we wouldn't run into issues?

Reply:
Apologies for the misunderstanding, I wanted to ask if the _Anniversary_ and _Newspaper of the Year_ logos could have the same size viewbox as the _Standard_ logo, and if not, could they have the same size viewbox as each other. I think you're right that `height: auto` should be safe and future-proof, especially if that's what we're already doing in DCR 💪 

@oliverlloyd does this sound okay to you?

Reply:
![grilla-yes](https://user-images.githubusercontent.com/1336821/130104042-06e8b21f-ee71-4821-a4ac-d091ae72254d.gif)